terraform/azure/projects/utoronto.tfvars

```diff
@@ -8,21 +8,39 @@ location                       = "canadacentral"
 storage_size = 8192
 ssh_pub_key  = "ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQDQJ4h39UYNi1wybxAH+jCFkNK2aqRcuhDkQSMx0Hak5xkbt3KnT3cOwAgUP1Vt/SjhltSTuxpOHxiAKCRnjwRk60SxKhUNzPHih2nkfYTmBBjmLfdepDPSke/E0VWvTDIEXz/L8vW8aI0QGPXnXyqzEDO9+U1buheBlxB0diFAD3vEp2SqBOw+z7UgrGxXPdP+2b3AV+X6sOtd6uSzpV8Qvdh+QAkd4r7h9JrkFvkrUzNFAGMjlTb0Lz7qAlo4ynjEwzVN2I1i7cVDKgsGz9ZG/8yZfXXx+INr9jYtYogNZ63ajKR/dfjNPovydhuz5zQvQyxpokJNsTqt1CiWEUNj georgiana@georgiana"
 
-# FIXME: upgrade to 1.27.7, and then 1.28.3, based on the latest versions
-#        available via: az aks get-versions --location westus2 -o table
-#
-kubernetes_version = "1.26.3"
-
-# FIXME: upgrade core_node_vm_size to Standard_E4s_v5
-core_node_vm_size = "Standard_E4s_v3"
-
-notebook_nodes = {
-  "default" : {
-    # NOTE: min-max below was set to 0-86 retroactively to align with
-    #       observed state without understanding on why 0-86 was picked.
+# List available versions via: az aks get-versions --location westus2 -o table
+kubernetes_version = "1.28.3"
+
+core_node_pool = {
+  name : "core",
+  kubernetes_version : "1.28.3",
+
+  # FIXME: transition to "Standard_E2s_v5" nodes as they are large enough and
+  #        can more cheaply handle being forced to have 2-3 replicas for silly
+  #        reasons like three calico-typha pods. See
+  #        https://github.com/2i2c-org/infrastructure/issues/3592#issuecomment-1883269632.
+  #
+  vm_size : "Standard_E4s_v3",
+
+  # FIXME: stop using persistent disks for the nodes, use the variable default
+  #        "Temporary" instead
+  kubelet_disk_type : "OS",
+
+  # FIXME: use a larger os_disk_size_gb than 40, like the default of 100, to
+  #        avoid running low when few replicas are used
+  os_disk_size_gb : 40,
+
+  # FIXME: its nice to use autoscaling, but we end up with three replicas due to
+  #        https://github.com/2i2c-org/infrastructure/issues/3592#issuecomment-1883269632
+  #        and its a waste at least using Standard_E4s_v3 machines.
+  enable_auto_scaling : false,
```

Comment:
I'm concerned that we don't have the capacity to fully deal with the fact that this would be the *only* nodepool in our entire fleet that doesn't have autoscaling turned on, and all the changes that entails. For example, I think this would mean that cordoning and draining the node won't automatically bring up another node, so I'm not exactly sure how we would deal with nods failing (that do happen). Given other constraints on our engineering capacity, I think we should avoid special casing unless we absolutely must.

So my suggestion is to make the cost trade-off here, and not try to optimize away the third node. Even if we want to optimize cost for UToronto, I think there are other avenues to tackle (like their huge home directory for example). Let's turn autoscaling back on, and we can deal with the extra node once upstream fixes their configurability issues.

Reply:
> I'm concerned that we don't have the capacity to fully deal with the fact that this would be the only nodepool in our entire fleet that doesn't have autoscaling turned on, and all the changes that entails.

I think you are right, this is just too strong of a point!